libraries/AP_HAL/board/chibios.h

```diff
@@ -86,6 +88,10 @@
 #define HAL_INS_ICM20608_AM_NAME "icm20608-am"
 #define HAL_INS_ICM20608_EXT_NAME "icm20608_ext"
 
+#ifndef HAL_INS_RATE_LOOP
+#define HAL_INS_RATE_LOOP 1
+#endif
+
```

Comment:
Maybe mve this default-to-1 code up into `AP_HAL/AP_HAL_Boards.h`?

Reply:
Fixed